### PR TITLE
Manage all kernel opt

### PR DESCRIPTION
**Pull request by edelangh**
fix issue https://github.com/Seed-Up/rust-fuse/issues/6

### CODE REVIEW

File: src/fuse.rs [hunk 1]
```diff
@@ -700,3 +701,37 @@ pub fn fuse_mount_compat25(mountpoint: &PathBuf, args: &fuse_args) -> std::io::R
...
+                        | "blkdev"
+                        | "blksize"
+                        | "max_read"
+                        | "fd"
+                        | "user_id"
+                        | "fsname"
+                        | "subtype" => {
+                    Some(String::from(opt))
+                }
+                _ => None
```
Comment (roblabla) [inline src/fuse.rs:731]: Shouldn't we return an error if we find an unknown option instead ?

**Comment by edelangh** [comment]
I think you're right but kernel, fuse hl, fuse ll, arg are melt together, and i didn't manage all of then at this time. 
Maybe I can create an issue to fix that after done all opt

File: src/fuse.rs (hunk 1 above)
Comment (edelangh) [reply src/fuse.rs:731]: I think you're right but kernel, fuse hl, fuse ll, arg are melt together, and i didn't manage all of then at this time.
Maybe I can create an issue to fix that after done all opt

Comment (roblabla) [reply src/fuse.rs:731]: OK that works

**Review by roblabla — APPROVED** [review]

**Pull request merged**